src/main/scala/cognite/spark/v1/fdm/FlexibleDataModelBaseRelation.scala

```diff
@@ -583,7 +585,13 @@ abstract class FlexibleDataModelBaseRelation(config: RelationConfig, sqlContext:
     }
 }
 
-object FlexibleDataModelBaseRelation extends NamedRelation {
+object FlexibleDataModelBaseRelation
+    extends NamedRelation
+    with UpsertSchema
+    with DeleteWithExternalIdSchema {
+  override val upsertSchema: StructType = StructType(
+    Seq(StructField("externalId", DataTypes.StringType)))
```

Comment:
I think it makes sense to put custom deleteSchema here with a comment that is potentially looking incorrect (missing space or even more fields)